### PR TITLE
Fix issues loading polyfills in IE11 & co.

### PR DESCRIPTION
**Pull request by DFurnes**
### What does this PR do?

This PR fixes issues rendering the application in IE11 and other older browsers. I'd tried to be a little too clever with the feature tests & they weren't properly pushing things into the `features` array. Now they do! I've also included `regenerator-runtime` in the vendor bundle so we can use Babel-ified async/await in our application code.

### Any background context you want to provide?
I remember this after taking a peek at [browser errors](https://rpm.newrelic.com/accounts/108038/browser/68967300/js_errors#id=772644512&tab_index=1) for the past day in New Relic!

### What are the relevant tickets/cards?

N/A

### Checklist

* [ ] Documentation added for new features/changed endpoints.
* [ ] Added appropriate feature/unit tests.
* [ ] Added screenshot to PR description of related front-end updates on **small** screens.
* [ ] Added screenshot to PR description of related front-end updates on **medium** screens.
* [ ] Added screenshot to PR description of related front-end updates on **large** screens.

### CODE REVIEW

**Comment by DFurnes** [comment]
Another note - I switched out the ["default" polyfill set](https://polyfill.io/v2/docs/features/#default-sets) for the entirety of ES6 and ES7 when a browser fails our minimal feature test. I'd made the (wrong) assumption that `Array.includes` was in the default set... it's not! Let's make that easy.

And a handy reference - check the [`polyfills`](https://github.com/Financial-Times/polyfill-service/tree/master/packages/polyfill-library/polyfills) directory in the [polyfill.io](https://polyfill.io/) source for `config.json` files to see what features are included in different bundles (and their browser support and dependencies)!

**Comment by DFurnes** [comment]
A real beaut!

screen shot 2018-05-02 at 11 52 59 am

And modern browsers (in this case, Safari 11.1) don't download anything extra:

screen shot 2018-05-02 at 11 56 41 am

File: resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -40,18 +40,23 @@
...
 
     <script type="text/javascript">
+        var hasWorkingUrl = false; // https://git.io/vpzRA
+        try { var u = new URL('b', 'http://a'); u.pathname = 'c%20d'; hasWorkingUrl = u.href === 'http://a/c%20d'; } catch(e) {}
+
         var features = [];
-        ('fetch' in window) || features.push('fetch');
-        ('URL' in window) || features.push('URL');
+        if (!('Map' in window) || !('WeakSet' in window)) features.push('es2015')
+        if (!('includes' in Array.prototype)) features.push('es2016')
```
Comment (mendelB) [inline resources/views/layouts/master.blade.php]: Interesting - so adding the whole of `es2016`, wouldn't automatically cover `Map` and `WeakSet`?

Comment (DFurnes) [reply resources/views/layouts/master.blade.php]: `Map` & `WeakSet` are included in the `es2015` bundle, and `[].includes` in the `es2016` one!

**Pull request merged**